templates/master/00-master/gcp/files/opt-libexec-openshift-gcp-routes-sh.yaml

```diff
@@ -64,6 +64,12 @@ contents:
         # balanced, even if the DNAT entry is removed
         ensure_rule filter INPUT -m comment --comment 'gcp LB vip existing' -m addrtype ! --dst-type LOCAL -m state --state ESTABLISHED,RELATED -j ACCEPT
 
+        # Don't allow GCP HealthCheckers traffic 130.211.0.0/22 and 35.191.0.0/16 to loop through the node
+        # HealthCheck traffic MUST go only to the INPUT chain, NEVER to the FORWARD chain
```

Comment:
This comment doesn't really help me understand what's going on. How is the traffic ending up in the FORWARD chain? Why is it important that it doesn't?

Reply:
This bug is complicated, let me try to do my best to explain it, the the problem is that there are network problems with the apiserver https://bugzilla.redhat.com/show_bug.cgi?id=192569 caused by stale conntrack entries created by the GCP healthcheckers.

Important things to know:
- The GCP healthcheckers poll the VMs with VIPs continuously from a well known source ranges.
- It seems that GCP install an agent in the VM that does the magic to handle the VIP logic.
- Openshift doesn't use the GCP agent, instead it uses this bash script that install redirect rules to the VIP
- However, to not redirect traffic to the local VIP if it is down, there is a pod (apiserver-watcher pod) that polls the apiserver and drops a file in a specific path so the bash script knows when to redirect traffic to the local apiserver or not
- Michael Gugino pinpointed the problem, the traffic from the healthchecker creates stale entries, if you delete those conntrack entries everything works https://bugzilla.redhat.com/show_bug.cgi?id=1925698#c29
- I was able to trace the stale entries, and are caused because the healthcheck TCP SYN is hairpining and it is always renewed, until you clean it https://bugzilla.redhat.com/show_bug.cgi?id=1930457#c3 
- I've implemented the conntrack solution, but when you delete the conntrack entries you affect other traffic, and I couldn't find any way in conntrack on saying "just delete the stale entries only" or "the entries from the healthcheckers"
- This PR now implements the solution suggested by Florian Westphal https://bugzilla.redhat.com/show_bug.cgi?id=1930457#c8 , that is just don't allow the healthcheck traffic to go to the FORWARD chain, because it should never be forwarded as is directed to the host.

I will add the reference to the bugs bz192569 and bz1930457 and try to summarise this in the comment.